partiql-planner/src/test/kotlin/org/partiql/planner/internal/typer/PlanTyperTestsPorted.kt

```diff
@@ -2506,37 +2494,27 @@ class PlanTyperTestsPorted {
             SuccessTestCase(
                 key = PartiQLTest.Key("basics", "case-when-17"),
                 catalog = "pql",
-                expected = unionOf(StaticType.CLOB, StaticType.NULL),
+                expected = StaticType.CLOB,
             ),
             SuccessTestCase(
                 key = PartiQLTest.Key("basics", "case-when-18"),
                 catalog = "pql",
-                expected = unionOf(StaticType.STRING, StaticType.NULL),
+                expected = StaticType.STRING,
             ),
             SuccessTestCase(
                 key = PartiQLTest.Key("basics", "case-when-19"),
                 catalog = "pql",
-                expected = unionOf(StaticType.STRING, StaticType.NULL),
+                expected = StaticType.STRING,
             ),
             SuccessTestCase(
                 key = PartiQLTest.Key("basics", "case-when-20"),
                 catalog = "pql",
-                expected = StaticType.NULL,
+                expected = StaticType.ANY,
             ),
             SuccessTestCase(
                 key = PartiQLTest.Key("basics", "case-when-21"),
                 catalog = "pql",
-                expected = unionOf(StaticType.STRING, StaticType.NULL),
-            ),
-            SuccessTestCase(
-                key = PartiQLTest.Key("basics", "case-when-22"),
-                catalog = "pql",
-                expected = unionOf(StaticType.INT4, StaticType.NULL, StaticType.MISSING),
-            ),
-            SuccessTestCase(
-                key = PartiQLTest.Key("basics", "case-when-23"),
-                catalog = "pql",
-                expected = StaticType.INT4,
```

Comment:
Any reason these tests are deleted?

Reply:
Yeah, these tests originally are as follows:
```
--#[case-when-22]
-- type: (null|missing|int32)
CASE t_item.t_string
    WHEN 'a' THEN t_item.t_absent
    ELSE -1
END;

--#[case-when-23]
-- type: int32
-- false branch is pruned
CASE
    WHEN false THEN t_item.t_absent
    ELSE -1
END;
```

Since the t_absent type is now just ANY instead of `["null", "missing"]`, the result is just ANY -- and we have tests that already test this. That being said, I'm going to go one step further and delete the Ion tests and remove `t_absent` as well as `t_null` and `t_missing`.